docs/spark/tutorials/batch-processing.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Batch processing with .NET for Apache Spark tutorial
+description: Learn how to do batch processing using .NET for Apache Spark.
+author: mamccrea
+ms.author: mamccrea
+ms.date: 12/04/2019
+ms.topic: tutorial
+---
+
+# Tutorial: Do batch processing with .NET for Apache Spark
+
+In this tutorial, you learn how to do batch processing using .NET for Apache Spark. Batch processing is the transformation of data at rest, meaning that the source data has already been loaded into data storage. 
+
+Batch processing is generally performed over large, flat datasets that need to be prepared for further analysis. Log processing and data warehousing are common batch processing scenarios. In this scenario, you analyze information about GitHub projects, such as the number of time different projects have been forked or how recently projects have ben updated. 
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> * Create and run a .NET for Apache Spark application
+> * Read data into a DataFrame and prepare it for analysis
+> * Process the data using Spark SQL
+
+## Prerequisites 
+
+If this is your first time using .NET for Apache Spark, check out the [Get started with .NET for Apache Spark](../tutorials/get-started.md) tutorial to learn how to prepare your environment and run your first .NET for Apache Spark application.
+
+## Download the sample data
+
+[GHTorrent](http://ghtorrent.org/) monitors all public GitHub events, such as info about projects, commits, and watchers, and stores the events and their structure in databases. Data collected over different time periods is available as downloadable archives. Because the dump files are very large, this guide uses a [truncated version of the dump file](https://github.com/dotnet/spark/tree/master/examples/Microsoft.Spark.CSharp.Examples/Sql/Batch/projects_smaller.csv) that can be downloaded from GitHub.
```

Comment:
The PR that uploads this truncated file hasn't been merged yet, so this link wouldn't work today (but it will work once that PR is merged and the data is merged into the main .NET for Spark repo). So we'd just want to make sure this doc gets published when/after that PR is merged.

Reply:
Also, per [GHTorrent guidelines](http://ghtorrent.org/faq.html), we might need to include licensing information like the following:

`The GHTorrent dataset is distributed under a dual licensing scheme ([Creative Commons +](https://wiki.creativecommons.org/wiki/CCPlus)). For non-commercial uses (including, but not limited to, educational, research or personal uses), the dataset is distributed under the [CC-BY-SA license](https://creativecommons.org/licenses/by-sa/4.0/).`